### PR TITLE
Make stack trace more readable

### PR DESCRIPTION
**Pull request by mlanin**
This PR comes from the desire to make this awesome tool even better! :)

In current version if stack is too long, table starts to resize and you have to scroll the whole table to read or close opened stack. Now only stack row is scrollable.

Also switched to monospace font in trace, made it lighter, and added ability to highlight words in it. This behaviour is fully configurable and highlights line numbers by default.

### CODE REVIEW

**Comment by arcanedev-maroc** [comment]
Great work 👍 

There are some changes to be done but it looks awesome.

File: resources/views/_template/style.blade.php [hunk 1]
```diff
@@ -83,15 +83,23 @@
     .table-condensed > thead > tr > td.stack {
         padding: 0;
         border-top: none;
+        background-color: #F6F6F6;
+        border-top: 1px solid #D1D1D1;
+        max-width: 0;
+        overflow-x: scroll;
```
Comment (arcanedev-maroc) [inline resources/views/_template/style.blade.php]: By adding `overflow-x: scroll;`, the scrollbar is showing:

scrolls

Is that necessary ?

File: src/Contracts/Utilities/LogStyler.php [hunk 2]
```diff
@@ -31,4 +31,13 @@ public function icon($level, $default = null);
...
     public function color($level, $default = null);
+
+    /**
+     * Get strings to highlight.
+     *
+     * @param  array  $default
+     *
+     * @return string
+     */
+    public function toHighlight($default = []);
```
Comment (arcanedev-maroc) [inline src/Contracts/Utilities/LogStyler.php]: Can you add the `array` typehint to the `$default` argument  ?

`public function toHighlight(array $default = []);`

File: src/Utilities/LogStyler.php [hunk 3]
```diff
@@ -82,4 +82,16 @@ public function color($level, $default = null)
...
     }
+
+    /**
+     * Get strings to highlight.
+     *
+     * @param  array  $default
+     *
+     * @return string
+     */
+    public function toHighlight($default = [])
```
Comment (arcanedev-maroc) [inline src/Utilities/LogStyler.php]: Same thing here: `public function toHighlight(array $default = [])`

File: tests/Utilities/LogStylerTest.php [hunk 4]
```diff
@@ -101,4 +101,10 @@ public function it_can_use_helper_get_color()
             $this->assertHexColor(log_styler()->color($level));
         }
     }
+
+    /** @test */
+    public function it_can_get_string_to_highlight()
+    {
+        $this->assertCount(2, $this->styler->toHighlight());
```
Comment (arcanedev-maroc) [inline tests/Utilities/LogStylerTest.php]: Can you replace this line by:

```php
$expected = [
    '^#\d+',
    '^Stack trace:',
];

$this->assertSame($expected, $this->styler->toHighlight());
```

File: resources/views/show.blade.php [hunk 5]
```diff
@@ -193,6 +193,16 @@
 
                 return false;
             });
+
+            @if (!empty(log_styler()->toHighlight()))
```
Comment (arcanedev-maroc) [inline resources/views/show.blade.php]: Can you replace the `@if` directive by `@unless` ?

Something like this:

```blade
@unless (empty(log_styler()->toHighlight()))
...
@endunless
```

File: resources/views/_template/style.blade.php (hunk 1 above)
Comment (mlanin) [inline resources/views/_template/style.blade.php]: Strange.. what browser?

Comment (arcanedev-maroc) [inline resources/views/_template/style.blade.php]: Chrome 58.0.3029 (Windows 8.1)

Comment (arcanedev-maroc) [inline resources/views/_template/style.blade.php]: Question: Why adding the scrollbar to the stack content ?

Comment (mlanin) [inline resources/views/_template/style.blade.php]: Understood the problem. I thought that TR itself is toggling, not only div with stack.

**Comment by mlanin** [comment]
Fixed scroll

**Comment by mlanin** [comment]
What do you mean "BC Checks"?

**Comment by arcanedev-maroc** [comment]
**BC**: **B**ackward **C**ompatibility

**Comment by mlanin** [comment]
Understood. There is nothing to be broken. The only change is config, but it's only updated with new root option, so it will not break anything.

**Comment by arcanedev-maroc** [comment]
Merged ! Thanks a lot @mlanin 👍

**Comment by mlanin** [comment]
👍

**Comment by mlanin** [comment]
One more question. Why did you extracted installation of `orchestra/testbench` to scripts instead of requiring it in `require-dev`?

**Comment by arcanedev-maroc** [comment]
I use a *trick* to test the packages with multiple laravel versions.

Check this: https://github.com/ARCANEDEV/LogViewer/blob/4.2/.travis.yml#L15-L23

**Comment by mlanin** [comment]
Reasonable :)

**Pull request merged**